course/permissions.py

```diff
@@ -130,6 +130,9 @@ def is_object_visible(self, request, view, course):
         """
         # NOTE: course is actually course instance
 
+        if isinstance(request.user, GraderUser):
```

Comment:
What has changed compared to A+ v1.11? I assume that GraderUser was never before using this permission, so why is it now here?

Reply:
It was using it, the GraderUser was just always assumed to be a teacher, so it got skipped on the next if statement.

Reply:
So what happened if GraderUser is now not counted as a teacher?

Reply:
Being a teacher implies write permissions to a course, so we do not assume that the GraderUser is always a teacher. They are only counted as a teacher if they have write permissions to the course (listed in the JWT)

Reply:
Thanks for the clarification!